### PR TITLE
Add support for Ubuntu 18.04 / systemd to Mendel 

### PR DESCRIPTION
**Pull request by Ibracadabra05**
- Add service wrapper command to startup services, which is also backward compatible with ubuntu 14.04.
- Add scripts to startup and gracefully shutdown systemd container before and after integration tests, respectively.
- Add end to end integration tests to validate deployments to [systemd container](https://cloud.docker.com/u/ihamisu/repository/docker/ihamisu/heavyset).

For more context: https://sprout.atlassian.net/browse/PLATTOOL-67

All integration tests pass:

screen shot 2019-02-05 at 2 30 40 pm

### CODE REVIEW

File: mendel/core.py [hunk 1]
```diff
@@ -119,6 +119,7 @@ def __init__(
             slack_url=None,
             slack_emoji=":rocket:",
             use_upstart=True,
```
Comment (Ibracadabra05) [inline mendel/core.py]: Leaving this so people who still have it in their `mendel.yml` don't get unnecessary errors. They'll just have to bump their mendel version and switch to `use_init` when they're ready to switch over to `ubuntu 18.04` servers.

**Comment by stahs** [comment]
@Ibracadabra05 is there any documentation on these tox integration tests? I just ran `tox` in the working directory (after installing requirements) and it failed with a ton of ominous looking messages.

File: mendel/core.py (hunk 1 above)
Comment (stahs) [reply mendel/core.py]: Perhaps add a deprecation warning message. If someone is running this version of Mendel, they should be using the correct flag for init.

**Comment by Ibracadabra05** [comment]
@stahs some can be found [here](https://sprout.atlassian.net/wiki/spaces/INFRA/pages/568492133/Remote+jar+deployments+in+Mendel#RemotejardeploymentsinMendel-Setup) but I suspect what you lack is
```
<server>
      <id>docker_container</id>
      <username>admin</username>
      <password>admin123</password>
</server>
```
within the settings tag in your `~/.m2/settings.xml` file.

**Comment by stahs** [comment]
Test deployment of twitter-sup-dude successful - very nice!

**Comment by Ibracadabra05** [comment]
@stantonk gonna need another 👍 when you're chanced?!

**Pull request merged**